mkdocs_meta_descriptions_plugin/plugin.py

```diff
@@ -18,6 +19,7 @@ class MetaDescription(BasePlugin):
         ("enable_checks", config_options.Type(bool, default=False)),
         ("min_length", config_options.Type(int, default=50)),
         ("max_length", config_options.Type(int, default=160)),
+        ("trim", config_options.Type(bool, default=False))
```

Comment:
I suggest keeping the trailing comma after the last item in the tuple, to [make future maintenance easier](https://stackoverflow.com/a/7992643).

```suggestion
        ("trim", config_options.Type(bool, default=False)),
```